File: Python/CountSubArraysWithMajorityElement_I.py
class CountSubArraysWithMajorityElement_I(object):
    def countMajoritySubarrays(self, nums, target):
        n = len(nums)
        ans = 0

        for i in range(n):
            count = 0
            for j in range(i, n):
                if nums[j] == target:
                    count += 1

                if count > (j - i + 1) // 2:
                    ans += 1

        return ans

File: JS/CountSubArraysWithMajorityElement_I.js
var countMajoritySubarrays = function(nums, target) {
    const n = nums.length;
    const ans = 0;
    for(let i=0; i<n; i++) {
        const count = 0;
        for(let j=i; j<n; j++) {
            if(nums[j]==target) count++;
            if(count > (j-i+1)/2) ans++;
        }
    }
    return ans;
};